app/konnect/org-management/deactivation.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Account and Organization Deactivation
+no_version: true
+---
+
+A free {{site.konnect_saas}} organization is automatically deactivated after 30
+days of inactivity.
+
+Your organization is considered inactive when:
+* There is no user login into the organization during the past 30 days.
+* There are no API requests in either the current or the previous billing cycle
+(30 day increments).
+
+Organizations in Plus or Enterprise tiers are exempt, and are never deactivated
+automatically.
+
+## What happens if an organization is deactivated?
+
+If your org is deactivated, you can no longer log into the organization, either
+through the Konnect UI or the API, and the following happens:
+
+* All billing stops immediately, and all {{site.konnect_saas}} subscriptions
+are removed.
+
+* The control plane associated with the organization is decommissioned.
+
+* Any users that were part of the organization lose any roles or permissions
+associated with the organization. Their accounts are otherwise unaffected.
+
+* The email associated with the organization is locked and can't be used to
+create another {{site.konnect_saas}} account.
+
+If you have any runtimes registered through the Runtime Manager, they won't be
+stopped by {{site.konnect_saas}}. They will no longer proxy data, but the
+instances will keep running until you stop them manually.
+
+## Deactivate or reactivate an organization
+
+There is currently no way to manually deactivate an organization.
```

Comment:
```suggestion
To deactivate an organization that you registered, please contact [Kong Support](https://support.konghq.com/).
```

I suggest this rather than leaving the user at a dead end. If it is phrased this way because Customer Support doesn't want to field these requests, feel free to leave as is.

Reply:
I haven't heard complaints from support yet, and we have links to contact them all over the Konnect docs 😄 I'll add this.